src/core/mac/mac.cpp

```diff
@@ -966,7 +941,7 @@ void Mac::ProcessTransmitSecurity(TxFrame &aFrame, bool aProcessAesCcm)
 
     if (aProcessAesCcm)
     {
-        ProcessTransmitAesCcm(aFrame, extAddress);
+        aFrame.ProcessTransmitAesCcm(*extAddress);
```

Comment:
It seems to be possible that `extAddress` being `NULL` here, for example, in case `kKeyIdMode2`. Do we need to add a check here?

Reply:
Is it `null` in `kKeyIdMode2`?